container_registry/ec2_instance.tf

```diff
@@ -0,0 +1,77 @@
+data "aws_subnet" "quay_subnet" {
+  id = data.aws_subnets.unity_private_subnets.ids[1]
```

Comment:
Just a note of warning that I had an issue when script was picking the subnet from the list which was not corresponding to the AZ used by the LB. So I found that just listing all available subnets and then picking the one by index from the list could be error prone. I didn't find the best solution yet for the Dockstore deployment, but it should be picking up the subnet based on the specific AZ. Might not be relevant here?

Reply:
Indeed it is a problem. I have the same problem in the Jupyterhub deployment. It is probably best solved by using SSM keys provided by U-CS.